src/persist-client/src/cfg.rs

```diff
@@ -221,11 +221,12 @@ impl PersistConfig {
     }
 }
 
-pub(crate) const MB: usize = 1024 * 1024;
+#[allow(non_upper_case_globals)]
+pub(crate) const MiB: usize = 1024 * 1024;
```

Comment:
I originally typed this up as MIB because of the lint, but I think it's less clear that way. wdyt?

Reply:
I like this more!